src/bahmni.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# Description taken throuhg a synthesis of the github 
+# description and their FAQ
+description: >
+  Bahmi is an Open Source hospital Management System focusing
+  on poor/underserved and public hospitals in the developing
+  world.
+  It's aimed to being a generic system which can be used for
+  multiple diseases and hospitals in different countries.
+license: AGPL-3.0
+name: Bahmni
+programmingLanguages:
+  - java
+  - javascript
+  - python
+  - shell
+  - groovy
+# See guideline on how to rate a project
+rating: 5
+repoUrl: "https://github.com/Bahmni"
+sdgs: [3]
+websiteUrl: "https://www.bahmni.org/"
+
+contributionGuidelinesUrl: "https://bahmni.atlassian.net/wiki/spaces/BAH/pages/116069555/Contributing+to+Bahmni"
```

Comment:
Seeing a white page when trying to directly access this URL. Looks like some JIRA issue.

Reply:
works for me 🤔 
Do you mind checking again to see whether it's a permanent issue or if they were just down for a bit?
Their entire documentation is on the same site, though.
Alternate link would be: https://bahmni.atlassian.net/wiki/spaces/BAH/pages/1310748/Developer+s+Guide but that one is less friendly to read

Not sure what else we could use…

Reply:
I can open and read it from my phone. Maybe a temporary issue indeed. Thanks